src/main/java/io/xstefank/wildfly/bot/PullRequestFormatProcessor.java

```diff
@@ -161,4 +163,27 @@ private List<Check> initializeChecks(WildFlyConfigFile wildflyConfigFile) {
 
         return checks;
     }
+
+    @PostConstruct
+    void construct() {
+        SKIP_FORMAT_COMMAND = Pattern.compile("@%s skip format".formatted(wildFlyBotConfig.githubName()), Pattern.DOTALL | Pattern.CASE_INSENSITIVE | Pattern.LITERAL);
```

Comment:
you can do this in the declaration of the field, no need for post construct method, and drop case insesitivity